CHANGELOG.md

```
@@ -1,3 +1,8 @@
+## [Unreleased]
+
+#### Fixed
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)